css/admin/dashboard/dashboard-trending.css

```diff
@@ -0,0 +1,72 @@
+.trending-title {
+  margin-top: 74px;
+  margin-bottom: 14px;
+}
+
+.trending {
+  margin-left: 196px;
+}
+
+.trending-header {
+  display: flex;
+  margin-left: 155px;
+  margin-bottom: 4px;
+}
+
+.trending-header > span {
+  color: var(--tph-text-secondary) !important;
+  font-size: 18px;
+}
+
```

Comment:
Instead of margin-left, consider giving .trending-header a fixed width and setting justify-content to space-between.